pkg/scheduler/schedulercache/cache_test.go

```diff
@@ -310,9 +328,10 @@ func TestAddPodWillConfirm(t *testing.T) {
 	tests := []struct {
 		podsToAssume []*v1.Pod
 		podsToAdd    []*v1.Pod
-
-		wNodeInfo *NodeInfo
-	}{{ // two pod were assumed at same time. But first one is called Add() and gets confirmed.
+		name         string
```

Comment:
why make `name` the third field? Makes more sense at the top. (I've seen you add it to the bottom also, which is fine.)